pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -51,6 +51,10 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(cr *api.PerconaServerMo
 		}
 
 		return clusterError, errors.Wrap(err, "dial:")
+	} else if !cr.Status.Replsets[replset.Name].Initialized {
+		// this can happended when cluster was initialized but status update failed
+		cr.Status.Replsets[replset.Name].Initialized = true
```

Comment:
should we also add lines line 43-48 here?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/558/commits/2d96cf9aa5411f6a791dd9a688a561f0f4ddd372